contracts/optimistic-ethereum/libraries/wrappers/Lib_SafeExecutionManagerWrapper.sol

```diff
@@ -322,6 +322,24 @@ library Lib_SafeExecutionManagerWrapper {
         );
     }
 
+    /**
+     * Performs a safe asdfasdfasdfasdf call.
+     */
+    function safeUPGRADE(
+        address _address,
+        bytes memory _code
+    )
+        internal
+    {
+        _safeExecutionManagerInteraction(
+            abi.encodeWithSignature(
+                "ovmUPGRADECODE(address,bytes)",
```

Comment:
I can't find this function anywhere. Assuming this is todo.

Reply:
ah good catch.  this was just a name change I forgot to reflect.  We should remember to replace these all with `abi.encodeWithSelector`, this is a classic footgun.